arango/formatter.py

```diff
@@ -854,44 +854,17 @@ def format_view(body: Json) -> Json:
     :return: Formatted body.
     :rtype: dict
     """
-    result: Json = {}
+    # Remove only unnecessary keys
+    if "error" in body:
+        body.pop("error")
+    if "code" in body:
+        body.pop("code")
     if "globallyUniqueId" in body:
-        result["global_id"] = body["globallyUniqueId"]
-    if "id" in body:
-        result["id"] = body["id"]
-    if "name" in body:
-        result["name"] = body["name"]
-    if "type" in body:
-        result["type"] = body["type"]
-    if "cleanupIntervalStep" in body:
-        result["cleanup_interval_step"] = body["cleanupIntervalStep"]
-    if "commitIntervalMsec" in body:
-        result["commit_interval_msec"] = body["commitIntervalMsec"]
-    if "consolidationIntervalMsec" in body:
-        result["consolidation_interval_msec"] = body["consolidationIntervalMsec"]
```

Comment:
Unfortunately, we can't go with this change, because it affects backwards compatibility. For example, users would expect to find `consolidation_interval_msec` instead of `consolidationIntervalMsec`. Unless we do a major release, IMO we can't do it this way.